lib/push.js

```diff
@@ -129,10 +129,26 @@ module.exports = function (options) {
 
       var files = wrench.readdirSyncRecursive('.');
       var excludedPaths = ['.build', '.git', '.whdist', '.push.zip'];
-      if (!custom)
-      {
+      if (!custom) {
         excludedPaths = excludedPaths.concat('package.json', 'node_modules', 'Gruntfile.js', 'libs', 'tasks', 'options');
       }
+      if (fs.existsSync('webhookconfig.json')) {
+        try {
+          var contents = JSON.parse(fs.readFileSync('webhookconfig.json', 'utf8'));
+        } catch(err) {
+          if (err.name === 'SyntaxError') {
+            console.log('Improperly formatted or empty webhookconfig.json'.red);
+          }
+        }
+        if (!contents) {
+          console.log('Unable to parse webhookconfig.json'.red);
+        }
+        if (contents.pathsToExclude) {
+          for (var i = 0; i < contents.pathsToExclude.length; i++) {
```

Comment:
should validate this is a list before doing anything and give a nice warning otherwise